test/verify/check-machines-dbus

```diff
@@ -830,8 +830,8 @@ class TestMachinesDBus(machineslib.TestMachines):
                 source={"host": "127.0.0.1", "source_path": target_iqn}
             ).execute()
 
-            # iscsi-direct pool type is available since libvirt 4.7 which is not available yet in RHEL
-            if m.image not in ["rhel-8.0", "rhel-8.1"]:
```

Comment:
rhel-8-0 and rhel-8-1 should both be skipped because of insufficient libvirt version. This iscsi-direct subtests needs libvirt 4.7 which is not available in these two rhels.

You might see them succeeding but this is a cooincidence, because there is a known issue for iscsi on rhel-8-1, which makes the failed test skipped.

Reply:
Thanks! Fixed.